src/pages/iv-materials-tfr/index.js

```diff
@@ -0,0 +1,265 @@
+import { useContext } from 'react'
+import toast from 'react-hot-toast'
+import Table from 'src/components/Shared/Table'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useResourceQuery } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useWindow } from 'src/windows'
+import { ControlContext } from 'src/providers/ControlContext'
+import { useError } from 'src/error'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { getStorageData } from 'src/storage/storage'
+import RPBGridToolbar from 'src/components/Shared/RPBGridToolbar'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+import MaterialsTransferForm from './Form/MaterialsTransferForm'
+
+const IvMaterialsTransfer = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const { stack: stackError } = useError()
+  const { stack } = useWindow()
+
+  async function fetchGridData(options = {}) {
+    const { _startAt = 0, _pageSize = 50, params } = options
+
+    const response = await getRequest({
+      extension: InventoryRepository.MaterialsTransfer.qry,
+      parameters: `_filter=&_size=30&_startAt=${_startAt}&_sortBy=recordId desc&_pageSize=${_pageSize}&_params=${params || ''}`
+    })
+
+    if (response && response?.list) {
```

Comment:
no need for this if condition, u are already checking it by using response?.list?.map